test/tf_exec.cpp

```diff
@@ -472,13 +483,94 @@ TEST(tf_exec, Op_Rsqrt) {
 
   TF_CHECK_OK(session.Run({r}, &outputs));
 
-  ASSERT_EQ(outputs[0].shape(), tf::TensorShape({2,2}));
+  ASSERT_EQ(outputs[0].shape(), tf::TensorShape({2, 2}));
 
   auto mat = outputs[0].matrix<float>();
-  EXPECT_FLOAT_EQ(1.f/16.f, mat(0, 0));
-  EXPECT_FLOAT_EQ(1.f/4.f, mat(0, 1));
-  EXPECT_FLOAT_EQ(1.f/2.f, mat(1, 0));
-  EXPECT_FLOAT_EQ(1.f/8.f, mat(1, 1));
+  EXPECT_FLOAT_EQ(1.f / 16.f, mat(0, 0));
+  EXPECT_FLOAT_EQ(1.f / 4.f, mat(0, 1));
+  EXPECT_FLOAT_EQ(1.f / 2.f, mat(1, 0));
+  EXPECT_FLOAT_EQ(1.f / 8.f, mat(1, 1));
+}
+
+template <typename T>
+void approx_matrix_compare(typename tf::TTypes<T>::Matrix mat,
```

Comment:
Can you not use AssertTensorEquals(...)? 
Also - we use CamelCase for functions and snake_case for variable names.

Reply:
did not notice that function(AssertTensorEquals). will use it.